developer/installation/installation-windows.md

```diff
@@ -0,0 +1,124 @@
+# Installation for Windows
+
+## Install prerequisites:
+
+**Note: These commands all assume you are running them from an administrator shell**
+
+### Install node
```

Comment:
Style: `Node` and `Node.js` are capitalized.

`### Install Node`

Reply:
Many users may have Node already, but an older version. We should:

1. Show how to check if you have Node already, like this: https://www.gatsbyjs.org/tutorial/part-one/#check-your-development-environment

2. Specify what version of Node and we require.

3. Also... does Node.js include npm? If not, they need to also install npm right?